recipe/meta.yaml

```diff
@@ -8,12 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8e2969797627c8a663575c2fc6fcb53a05e37cdb83ee65f341fc6e0c3d0ced16
+  sha256: 15b2cc15f95a476fbe0623fd8f771e1e771310bf7a01f95412a0b605b6e47510
 
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt
-  noarch: python
+  skip: True  # [py<37]
 
 requirements:
   host:
```

Comment:
- Missing `pip` package in requirements
- Missing `wheel` package in requirements. We need it to place in `host` explicitly

Reply:
After looking in the requirements, `pip` and `wheel` are not explicitly mentioned in the upstream, that is why they were left out.